db/schema.rb

```diff
@@ -11,7 +11,17 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
-ActiveRecord::Schema.define(version: 20150823122126) do
+ActiveRecord::Schema.define(version: 20151011163832) do
+
+  create_table "books", force: :cascade do |t|
+    t.string   "title"
+    t.string   "author"
+    t.string   "synopsis"
+    t.string   "image_url"
+    t.boolean  "available"
+    t.datetime "created_at", null: false
+    t.datetime "updated_at", null: false
+  end
```

Comment:
Would like to add a field which shows the person who took out the book. Would only be for the admin interface and for when someone is taking out a book with the app.